wordpress-mobile/WordPress-iOS
# Testing: Add publish tests using the block editor

This PR adds a new Gutenberg test suite with two publish flow tests using the block editor. These are the same publish tests we already had running for Aztec (which are now in a single Aztec test suite).

This PR also switches the main navigation test to use the block editor instead of Aztec. (The navigation itself is the same, but we have to define which editor we expect to open after tapping the editor button in the bottom nav bar.)

Note: For now, the block editor tests only run in English. We'll need to look at either adding test IDs to Gutenberg or making the localizable strings from Gutenberg accessible to these tests (less stable) to run them in other device languages.

To test:
* Pull the latest mobile secrets, or create a file with test credentials at `~/.mobile-secrets/iOS/WPiOS/WPUITestCredentials.swift` (based on the template at `/WordPressUITests/WPUITestCredentials-Template.swift`).
* Open the simulator and confirm the device language is set to Ennglish.
* Run the all test suites except `LoginTests` in the `WordPressUITests` target.

Update release notes:
